src/Monolog/Handler/HipChatHandler.php

```diff
@@ -93,6 +106,22 @@ public function __construct($token, $room, $name = 'Monolog', $notify = false, $
         $this->room = $room;
         $this->format = $format;
         $this->host = $host;
+
+        $this->setVersion($version);
+    }
+
+    /**
+     * Convenience method for setting the API version. Valid values are
+     * - HipChatHandler::API_V1
+     * - HipChatHandler::API_V2
+     * @param $version string the API version to use
+     */
+    public function setVersion($version) {
```

Comment:
I don't think it is either. If the additional `setVersion()` method is an issue I'd just say drop it and then people will have to set all arguments...